Pythonwin/pywin/dialogs/list.py

```diff
@@ -6,12 +10,20 @@
 
 
 class ListDialog(dialog.Dialog):
-    def __init__(self, title, list):
+    def __init__(
+        self,
+        title,
+        # Sequence rather than Iterable to avoid single-use iterable
+        # TODO: Make the "list" parameter generic,
+        list: Sequence[Sequence[object]],
+    ):
         dialog.Dialog.__init__(self, self._maketemplate(title))
         self.HookMessage(self.on_size, win32con.WM_SIZE)
         self.HookNotify(self.OnListItemChange, commctrl.LVN_ITEMCHANGED)
         self.HookCommand(self.OnListClick, win32ui.IDC_LIST1)
-        self.items = list
+        # This subclass breaks LSP by making "items" not a method
+        # TODO: Consider making this a method or changing the base class to a property
+        self.items: Sequence[Sequence[object]] = list  # type: ignore[assignment]
```

Comment:
This is revealing an easy source of bugs where some Dialogs use `.items()` and some Dialogs use `.items`